backend/.env.example

```diff
@@ -1,11 +0,0 @@
-# Stage to run the application as [dev|test|prod]
```

Comment:
Dont delete this please

Reply:
didnt touch the .env example not sure what to change

Reply:
you deleted it in your first commit view training draft lol

Reply:
I have undeleted it